src/System.Utf8String.Experimental/tests/System/Utf8SpanTests.Comparison.cs

```diff
@@ -88,7 +88,7 @@ public static void Equals_NonOrdinal(string str1, string str2, StringComparison
                 return RemoteExecutor.SuccessExitCode;
             };
 
-            if (culture != null && PlatformDetection.IsUap) // need to apply a culture to the current thread
+            if (culture != null)
```

Comment:
This isn't correct.  The whole if block can be deleted now; the only reason for the remote invoke was UAP.  And then since there's only the one call site to the delegate, the code from the delegate can be extracted and the delegate removed.